index.js

```diff
@@ -303,6 +305,8 @@ export default class ModalSelector extends React.Component {
         );
     }
 
+
+
```

Comment:
Please also revert extra lines.  Try to keep the changes isolated to the functional changes.